cves/CVE-2013-0868.yml

```diff
@@ -1,368 +1,406 @@
+  answer: true
+subsystem:
+  question: |
+    What subsystems was the mistake in?
+
+    Look at the path of the source code files code that were fixed to get
+    directory names. Look at comments in the code. Look at the bug reports how
+    the bug report was tagged.
+
+    In FFmpeg, the Component field is useful here. Often people will say
+      "undetermined" - do more homework than them and make a judgement call if
+      you can.
+
+    Note: a filepath is NOT a subsystem - that's too granular.
+
+    A non-exhaustive examples for ffmpeg are:
+      * avcodec
+      * avdevice
+      * avfilter
+      * avformat
+      * avutil
+      * build system
+      * documentation
+      * fate
+      * ffplay
+      * ffprobe
+      * postproc
+      * swresample
+      * swscale
+      * tools
+      * trac
+      * website
+
+    If this involves fixing multiple subsystems, you can make this an array.
+
+    In 'answer', explain how you arrived at this determination.
+  answer: |
+    This vulnerability was found in the Huffyuv decoder. This is one of many
+    decoders that make up avcodec, the generic encoding/decoding framework.
+  name: avcodec
+interesting_commits:
+  question: |
+    Optional: are there any interesting commits between your VCC(s) and fix(es)?
+
+    Write a brief (under 100 words) description of why you think this commit was
+    interesting in light of the lessons learned from this vulnerability. Any
+    emerging themes?
+
+    This is a catch-all for any interesting commit you find.
+    Example of interesting commits:
+      * Giant refactors
+      * Changing an API or big dependency
+      * A change that looks really sketchy
+      * Commits where a developer mentions security in their message or comments
+  commits:
+  - commit: 2ca5ca2957607cf9604cc6068a9ebc898ff37b06
+    note: Large refactor which split codecs into distinct sections. Interesting as it shows that no one retested functionality or seemed to check security, rather they assumed it was safe.
+  - commit:
+    note:
+i18n:
+  question: |
+    Was the feature impacted by this vulnerability about internationalization
+    (i18n)? An internationalization feature is one that enables people from all
+    over the world to use the system. This includes translations, locales,
+    typography, unicode, or various other features.
+
+    Answer should be boolean. Write a note about how you came to the conclusions
+    you did.
+  answer: false
+  note: This is not an i18n issue, it's an input validation issue.
+ipc:
+  question: |
+    Did the feature that this vulnerability affected use inter-process
+    communication? IPC includes OS signals, pipes, stdin/stdout, message
+    passing, and clipboard. Writing to files that another program in this
+    software system reads is another form of IPC.
+
+    Answer should be boolean.
+  answer: true
+  note: The input data comes in over IPC, and decoded data is written out to another file or stream.

```

Comment:
I think by this argument just about anything would be considered IPC - so if the feature is not directly working with IPC features I would say no.

Reply:
Makes sense! I felt that this was a bit broad so thanks for clarifying this.